sentry/sentry.conf.example.py

```diff
@@ -209,13 +209,24 @@
     }
 )
 
+# Hacky stuff
```

Comment:
🤔 I'd be scared of this causing an issue in the future if we were to convert these into real options.

I'm also don't actually see how this works at all.

```python
>>> class A(object):
...   def __get__(self, instance, owner):
...     return 'foo'
...
>>> foo = A()
>>> foo
<__main__.A object at 0x10ca97ef0>
>>> str(foo)
'<__main__.A object at 0x10ca97ef0>'
```

Reply:
>I'd be scared of this causing an issue in the future if we were to convert these into real options.

I don't think we can drop support for the old configs easily. Even then, this would simply be harmless.

>I'm also don't actually see how this works at all.

The trick is, although we define these like module locals, they are actually bound to a dynamically created class instance: the `settings` instance from `django.conf`. So in reality, we are doing
```py
>>> class A(object):
...   def __get__(self, instance, owner):
...     return 'foo'
...
>>> class B(object):
...  foo = A()
>>> bar = B()
>>> bar.foo
```

which then triggers `__get__` and makes this work (tested locally already).

Reply:
lol dang, I see. Yeah. I'd still like to see this just done inside the initializer rather than this hack as a one-off for onpremise.